per/drf_views.py

```diff
@@ -701,6 +717,21 @@ class Meta:
             "appeal_code__region": ("exact", "in"),
         }
 
+    def get_cache_response(self, queryset, name, value):
+        if ops_learning_cache_response := OpsLearningCacheResponse.objects.filter(id=value).first():
```

Comment:
Let's also add additional check
```suggestion
        if value and (ops_learning_cache_response := OpsLearningCacheResponse.objects.filter(id=value).first()):
```